### ISSUE TITLE
[bug] failed to initialize properly (0xc000001d)

### ISSUE THREAD

**Issue by youxiaojie** [Feodor2/Mypal68#164]
fresh installed xp /sp3, extract mypal, and run, then a dialogue box:
mypal.exe - Application Error

the application failed to initialize properly (0xc000001d). Click on OK to terminate the application.

**Comment by FlorianisonGitHub** [comment]
68.12b and 68.13.7b does not support processors without SSE2.

**Comment by FlorianisonGitHub** [comment]
68.14b will arrive support without SSE and SSE2 soon.

**Comment by Feodor2** [comment]
Nobody cares I see - no new sse version

**Comment by Feodor2** [comment]
This is about next version
We have one vote

**Comment by FlorianisonGitHub** [comment]
68.14b should support non-sse/sse and windows 2000 without extended kernel (including xp rtm/xp sp1/xp sp2)

**Comment by youxiaojie** [comment]
> 68.14b should support non-sse/sse and windows 2000 without extended kernel (including xp rtm/xp sp1/xp sp2)

great!

**Comment by FlorianisonGitHub** [comment]
> > This is about next version We have one vote
> 
> 
> 
> where is vote?

That is what I voted for.

**Comment by Feodor2** [comment]
Not enough so far

**Comment by Feodor2** [comment]
2 from msfn, 4 so far

**Comment by AstroSkipper** [comment]
I don't know where I can vote for a new **SSE** release but here is my vote: [![ssupercool2.gif](https://i.postimg.cc/zBP1TrbD/ssupercool2.gif)](https://postimg.cc/Hc5KCF9F)

**Comment by Feodor2** [comment]
The voting is here, so let it be the next version.

**Comment by talvisota** [comment]
i am interested in more SSE versions - using real dual pentium iii machine

**Comment by youxiaojie** [comment]
no sse. runing from pentium. amd k6-2

**Comment by enigma131** [comment]
I have also an old computer with Athlon Xp and 1.25 Go Ram, If MyPal 68 had SSE version, can it run on it ?

**Comment by rocky7717** [comment]
Would definitely like to see a NEW version!!

**Comment by Shaknunic** [comment]
I vote for SSE support. My processor support it and I like to use Mypal68 besides Windows XP in modern systems like Windows 8.1 and 10. Even on Windows XP it makes the browser be a bit more lightweight when It does have SSE. I mean It's up to the dev, but if Mypal68 drops support, I'll just move on to a different browser that supports it.

**Comment by Joegan-2** [comment]
Would love a non-SSE2 version for use with Windows 2000 (at least with the extended kernel), as there are very few options for basic web browsing that still work with something like an Athlon XP.

**Comment by AstroSkipper** [comment]
I also have another computer with an **Athlon XP** processor. Once the current problems have been solved, I would be very happy to have an **SSE** version again.

**Comment by Feodor2** [comment]
Okey next version will be

**Comment by SleakaJ** [comment]
I would also like to see SSE versions of Mypal68.

If it's a lot of trouble to make them then maybe every second or third release?

**Comment by TiredOfWindows** [comment]
I sure could use the sse version. The latest one that you released works good on most sites but is a little slow. Are there that many sse2 computers that will run WinXP without significant driver problems? seems like Win7 would be a better choice! Keep up the good work though. I'm glad someone is taking an interest in WinXP. I'd like to be able to use mine online again! Side note: I found a site where you can get all the updates for old windows versions 95 and up using IE4 and up. 
(http://windowsupdaterestored.com)

**Comment by IP-CAM** [comment]
![swisscom_warning](https://github.com/user-attachments/assets/386fe08e-7da0-4bff-aac7-de56986b473a)

**Comment by TiredOfWindows** [comment]
Don't know about viruses. I used it on my Win98SE computer and scanned it from a Win10 computer and no viruses were found. I couldn't scan my WinXP computer from my Win10 but There hasn't been any problems except that I can't find a virus program that will run on it without SSE2!!

**Comment by NS-Clone** [comment]
> and scanned it from a Win10 computer

there is some blahblahblah than win15 antiviruses
doens't have legacy dos/win95 etc viruses in bases
and can't detect it

**Comment by ExPressFromOlds** [comment]
Also voting for SSE-version, would like to try it on a monster Dual-Pentium III server that I've got. 

Tried earlier SSE-builds, and noticed, that the sites that worked good while accessed from the newer PCs with SSE2 suddenly stopped to work when the same browser was started on SSE1-only CPU - while the browser itself works, some scripts are clearly not running. More testing needed.

**Comment by wesinator** [comment]
The `-sse` relese is the version with SSE only?
It's a little smaller so I thought it might be _more_ optimized.

[NO_TRAIN]::

**Comment by Feodor2** [comment]
Just removed the sse2 and higher things from video and image decoders.
Also some misdone with images broke some colors waiting to be fixed.

**Comment by Laurpower721** [comment]
Also voting for SSE-version.

**Comment by Feodor2** [comment]
Does the latest version works on the pentium3?
I don't have it for check

**Comment by Feodor2** [comment]
Okey, please check 68.14.8.

**Comment by NS-Clone** [comment]
> Does the latest version works on the pentium3? I don't have it for check

ты наверно можешь проверить в qemu
там вроде даже 486 есть

**Comment by Feodor2** [comment]
> можешь проверить в qemu

я раз пробовал qemu - это хлам, зря потраченое время. Так и даже если нет, я делаю для реального железа и проверить нужно на нём.

**Comment by NS-Clone** [comment]
> я раз пробовал qemu - это хлам, зря потраченое время.

ну арм вин10 запускает и дажо со звуком

**Comment by MikePRoberts** [comment]
I also have nice pentium iii laptop. (Dell CSX) 498 Mhz, 512MB RAM.  Is there a version that will run on this?  With MyPal 68.14.4. en-US.win32 get error as start of this thread.  Thank you.  Mike

**Comment by Feodor2** [comment]
Yes, take the appropriate one (with the 'sse' thing in the name) and do not expect much, your nice laptop is a bare minimum.

**Comment by unclecaptain5426** [comment]
> 68.14b should support non-sse/sse and windows 2000 without extended kernel (including xp rtm/xp sp1/xp sp2)

I managed getting Mypal68 on Windows XP RTM (xpclient.010817-1148) without One Core API unofficially by using kernelxp.dll wrappers by roytam1, and swapped system files (kernel32.dll, advapi32.dll, shell32.dll, and shlwapi.dll) with the ones from XP SP1, and swapped sxs.dll with the one from XP SP3.

Still, no updated API-wrappers to have it to work on Windows 2000 without Extended Kernel yet.

**Comment by NS-Clone** [comment]
@Feodor2 кстате mypal неспешно взлетает вна intel sde
https://www.intel.com/content/www/us/en/download/684897/intel-software-development-emulator.html
может можно тестировать для старых процов не отходя от кассы

хотя там самые старые процы SSE3 и 1-й наверно? c SSE2
```
     -p4                 Set chip-check and CPUID for Pentium4
     -p4p                Set chip-check and CPUID for Pentium4 Prescott
```

хотя есть некий квак который по идеи должен быть вообще без ничаго (дажо без ММХ :rofl: )
```
     -quark              Set chip-check and CPUID for Intel(R) Quark
```

как минимум 7.31.0 если пропатчить PE работает на ХРю
но там интол страдает удолизмом
и старые версии в инторнетах практически не найти
при необходимости могу какието подкинуть

ну и возможно оно не все исключения выгребает

**Comment by NS-Clone** [comment]
хотя вот так 
```sde.exe -quark -emit-illegal-insts 0 -- Cpuz.exe```
cpuz тоже завелсо
и выдол такого зверя
<img width="643" height="192" alt="Image" src="https://github.com/user-attachments/assets/15ae03af-4e14-41d6-a14e-afb911c41f21" />

зотя там есть misc\cpuid\pentium3\cpuid.def
но не есно это именно cpuid или оно будет эмулиовать p3
и хз как запускать наверно так
```sde.exe -cpuid_in misc\cpuid\pentium3\cpuid.def```
cpuz с ним не взлетает

иге но удалось получить хотя бы в детектелке ``MMX:YES,SSE:YES,SSE2:NO``

**Comment by NS-Clone** [comment]
во а сейчас запустил
хотя даже вчера нагуглил что комбинировать команды с -cpuid_in нельзя
может то я пробовал на более старой версии

```
sde.exe -cpuid_in misc\cpuid\pentium3\cpuid.def -emit-illegal-insts 0 -- cpuz_x32.exe
```

<img width="393" height="189" alt="Image" src="https://github.com/user-attachments/assets/ffadbdf9-7d62-4d36-bd9b-0b2314b4ad9e" />

**Comment by TiredOfWindows** [comment]
Mypal 74.1.4 sse version is working well on my PIII computers. Seems to be pretty stable without crashes and the speed is much improved. Keep up the good work!
